metricbeat/module/couchbase/syncgateway/_meta/docs.asciidoc

```diff
@@ -0,0 +1,23 @@
+Sync Gateway is the synchronization server in a Couchbase for Mobile and Edge deployment. This metricset allows to monitor a Sync Gateway instance by using its REST API.
+
+Sync Gateway access `[host]:[port]/_expvar` on Sync Gateway nodes to fetch metrics data, ensure that the URL is accessible from the host where Metricbeat is running.
+
+[float]
+=== Skipping some events
+
+As mentioned before, this metricset generates 4 types of events under `couchbase.syncgateway.*` JSON path: DB stats, global stats, Memory stats and per replication stats. Only DB stats are always sent but you can choose to skip the others. For example, if you just want DB stats, you'll configuration will look like this:
+
+[source,yaml]
+----
+- module: couchbase
+  metricsets:
+    - syncgateway
+  period: 10s
+  hosts: ["myhost:4985"]
+  skip:
+    memstats: true
```

Comment:
maybe change to `mem_stats` instead?

Reply:
I'm gonna make a new module with its own metricset that I'll call `memory`. I have been renaming this back and forth from `memstats` which is the original name for this to `memory_stats` then `mem_stats` then back to `memstats` becasuse I can't decide lol